src/content/en/2021/media.md

```diff
@@ -1,20 +1,875 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=89579817&format=interactive",
+  sheets_gid="1030564653",
+  sql_file="image_srcset_descriptor.sql"
+  )
+}}
+
+How are developers populating their `srcset`s with resources?
+
+###### Number of candidates
+
+Let's first take a look at the number of candidate resources developers are including:
+
+{{ figure_markup(
+  image="number-of-srcset-candidates.png",
+  caption="Number of srcset candidates.",
+  description="A stacked bar chart showing the percentage of `srcset` attributes which contain different numbers of candidates. Almost all (80-90%) appear to contain 1-5 candidates, on both desktop and mobile. Next we have the 5-10 candidate bucket, which takes us up past 95%. Almost all of the remaining `srcset`s have 10-15 candidates, only a tiny sliver have 15-20.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=1875401927&format=interactive",
+  sheets_gid="1586096291",
+  sql_file="image_srcset_candidates.sql"
+  )
+}}
+
+A large majority of `srcset`s  are populated with five-or-fewer resources.
+
+###### Resource densities
+
+Are developers giving the browser an appropriately wide range of choices? To figure this out, we can calculate each resource's <a hreflang="en" href="https://html.spec.whatwg.org/multipage/images.html#current-pixel-density">density</a>: a measure of how many image pixels the `<img>` will paint in each CSS `px`, if left to its intrinsic dimensions. If the range of resource densities covers a reasonable range of real-world device DPRs, the `srcset` has a wide-enough range.
+
+{{ figure_markup(
+  image="distribution-of-image-srcset-candidate-densities.png",
+  caption="Distribution of image srcset candidate densities.",
+  description="A bar chart showing the distribution of image srcset candidate densities, on both desktop and mobile. Only the mobile bars (which are always higher) are labelled. At the tenth percentile, mobile candidates have a density of 0.7x; the desktop bar appears to be about 0.4x. At the 25th, 1.0x on mobile, and close to 0.8x on desktop. At the median (50th percentile), candidates on mobile have a density of 1.5x; the desktop density appears to be about 1x. At the 7th percentile, the mobile density is 2.7x and the desktop density appears to be about 2.0x. Lastly, at the 90th percentile, mobile candidates had 4.3x and desktop candidates were around 3.15x.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=403058700&format=interactive",
+  sheets_gid="1410495845",
+  sql_file="image_srcset_densities.sql"
+  )
+}}
```

Comment:
Every candidate has a density. That density possibly changes with context (resolved sizes value). This is a chart of the distribution of all of those densities, in each context.

This whole section requires a rather deep understanding of the srcset mechanism to understand, and, as I point out at the end, isn't really even as useful as it could be. Another iteration or two of analysis would have gotten me to a clear conclusion (srcsets are/aren't covering the ranges they need to). This is a step short, in a way that takes a lot of effort to explain/understand.

What would you think about deleting this whole section, rather than leading folks through a long-winded explanation of the srcset process, only to get to a conclusion that I wish was stronger?